pkg/kv/kvserver/replica.go

```diff
@@ -323,6 +323,12 @@ type Replica struct {
 		stateMachine replicaStateMachine
 		// decoder is used to decode committed raft entries.
 		decoder replicaDecoder
+
+		// bytesAccount accounts bytes used by various Raft components, like entries
+		// to be applied.
+		bytesAccount mon.BoundAccount
```

Comment:
nit: I think we usually use `mon`, `memMon`, or `memMonitor` for this.

Reply:
This is the "account" type which is a client of a monitor, so should we stick to `Account` in the name?